src/App.tsx

```diff
@@ -18,25 +18,37 @@ import { useAsync } from 'react-async-hook'
 import Portal from '@burstware/react-native-portal'
 import { ActionSheetProvider } from '@expo/react-native-action-sheet'
 import { BottomSheetModalProvider } from '@gorhom/bottom-sheet'
+import * as SplashScreen from 'expo-splash-screen'
 import { theme } from './theme/theme'
 import NavigationRoot from './navigation/NavigationRoot'
 import { useAppDispatch } from './store/store'
-import appSlice from './store/user/appSlice'
+import appSlice, { restoreUser } from './store/user/appSlice'
 import { RootState } from './store/rootReducer'
 import { fetchData } from './store/account/accountSlice'
 import BluetoothProvider from './providers/BluetoothProvider'
 import ConnectedHotspotProvider from './providers/ConnectedHotspotProvider'
 import * as Logger from './utils/logger'
 import { configChainVars, initFetchers } from './utils/appDataClient'
+import { fetchInitialData } from './store/helium/heliumDataSlice'
+import sleep from './utils/sleep'
+
+SplashScreen.preventAutoHideAsync()
 
 const App = () => {
   if (Platform.OS === 'android') {
     if (UIManager.setLayoutAnimationEnabledExperimental) {
       UIManager.setLayoutAnimationEnabledExperimental(true)
     }
-    LogBox.ignoreLogs(['Setting a timer'])
   }
 
+  LogBox.ignoreLogs([
```

Comment:
Oh nice, I added this in the index file too but left it commented out. Ill remove mine and we can just merge yours in